frontend/packages/metal3-plugin/src/selectors/machine.ts

```diff
@@ -0,0 +1,14 @@
+import { apiVersionForModel, MachineKind } from '@console/internal/module/k8s';
+import { compareOwnerReference } from '@console/kubevirt-plugin/src/utils';
+import { MachineSetModel } from '@console/internal/models';
+import { getOwnerReferences } from '@console/shared/src';
+
+export const getMachineMachineSetOwner = (machine: MachineKind) => {
```

Comment:
This would be beetter to add into @console/shared/src/selectors/machine.ts.

Reply:
I would rather move this in a followup once #4014 merges and make it more generic